### PR TITLE
Feature/versioning

### PR DESCRIPTION
**Pull request by RobjS**
Adds version info to `composer.json` so we can start using this as a dependency in wpc.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "name": "dxw/whippet",
     "description": "",
+    "version": "1.0.0",
```
Comment (mallorydxw) [inline composer.json]: https://getcomposer.org/doc/04-schema.md#version

> The version of the package. In most cases this is not required and should be omitted (see below).

Instead, add a `v1.0.0` git tag.

Comment (RobjS) [reply composer.json]: Good catch, I've removed that now. I'll tag the merge commit once we've got this merged, that somehow feels more correct to me than adding the tag to the last commit in this MR.

File: readme.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ During development, whippet can be used in conjunction with [wpc](https://github
 To install Whippet, clone this directory and install its dependencies:
 
 ```
-$ git clone --recurse-submodules https://github.com/dxw/whippet.git
+$ git clone https://github.com/dxw/whippet.git
```
Comment (erbridge) [inline readme.md]: 🎉

**Comment by mallorydxw** [comment]
@RobjS Are we going to get rid of the `dev-master` dependencies from `composer.json` in this PR?

```
        "dxw/rubbishthorclone": "dev-master",
        "mjackson/optionparser": "dev-master"
```

**Comment by RobjS** [comment]
@mallorydxw I think we could probably handle that in a separate MR. 

The `composer.json` files that will be including Whippet should only contain dependencies for running the site locally, rather than anything that will be running on production (as they'll be in the top level of the app, rather than in a theme folder), so I feel like downgrading their `minimum-stability` to `dev` isn't too much of a cause for concern.

**Pull request merged**